dev/drivers/scripts/narre/plots/jevs_narre_past90days_plots.sh

```diff
@@ -54,6 +28,10 @@ export RUN=atmos
 export VERIF_CASE=grid2obs
 export MODELNAME=narre
 
+module reset
+source $HOMEevs/modulefiles/$COMPONENT/${COMPONENT}_${STEP}.sh
+export MET_bin_exec=bin
```

Comment:
This line is no longer necessary due to the machine.conf having this information.